README.md

```diff
@@ -14,20 +14,11 @@ Takeaway Challenge
 
  ```
```

Comment:
General comments:

Your read me is really clear, all your tests are passing. when I ran restaurant.rb in IRB it worked well, however basket.rb threw some errors on total and print_order.